cli/tests/integration/mod.rs

```diff
@@ -1087,6 +1087,31 @@ fn typecheck_declarations_unstable() {
   assert!(output.status.success());
 }
 
+#[test]
+fn typecheck_core() {
+  let deno_dir = TempDir::new().expect("tempdir fail");
+  let test_file = deno_dir.path().join("test_deno_core_types.ts");
+  std::fs::write(
+    &test_file,
+    format!(
+      "import \"{}\";",
+      util::root_path()
```

Comment:
It's probably doing something like `D:\something\code/lib.deno_core.d.ts` on windows.